cli/commands/deposit/keys.go

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: BUSL-1.1
+//
+// Copyright (C) 2025, Berachain Foundation. All rights reserved.
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file of this repository and at www.mariadb.com/bsl11.
+//
+// ANY USE OF THE LICENSED WORK IN VIOLATION OF THIS LICENSE WILL AUTOMATICALLY
+// TERMINATE YOUR RIGHTS UNDER THIS LICENSE FOR THE CURRENT AND ALL OTHER
+// VERSIONS OF THE LICENSED WORK.
+//
+// THIS LICENSE DOES NOT GRANT YOU ANY RIGHT IN ANY TRADEMARK OR LOGO OF
+// LICENSOR OR ITS AFFILIATES (PROVIDED THAT YOU MAY USE A TRADEMARK OR LOGO OF
+// LICENSOR AS EXPRESSLY REQUIRED BY THIS LICENSE).
+//
+// TO THE EXTENT PERMITTED BY APPLICABLE LAW, THE LICENSED WORK IS PROVIDED ON
+// AN “AS IS” BASIS. LICENSOR HEREBY DISCLAIMS ALL WARRANTIES AND CONDITIONS,
+// EXPRESS OR IMPLIED, INCLUDING (WITHOUT LIMITATION) WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, NON-INFRINGEMENT, AND
+// TITLE.
+
+package deposit
+
+import (
+	"encoding/base64"
+
+	"github.com/berachain/beacon-kit/cli/context"
+	"github.com/berachain/beacon-kit/errors"
+	"github.com/berachain/beacon-kit/node-core/components"
+	"github.com/berachain/beacon-kit/node-core/components/signer"
+	"github.com/cometbft/cometbft/crypto/bls12381"
+	"github.com/spf13/cobra"
+)
+
+// GetValidatorKeysCmd returns a command that returns the validator public key in different formats
+// for the given private key files.
+//
+//nolint:lll // reads better if long description is one line.
+func GetValidatorKeysCmd() *cobra.Command {
+	cmd := &cobra.Command{
```

Comment:
@calbera we do have requests from backend to have a tool that helps mapping "Comet-like" pub keys to "Eth-like" pub keys.
Should we rework this to accept the either and return the other?

Reply:
this was more as a tool for validators that @codingwithmanny found to be useful. I think for backend they should use the underlying library and not rely on CLI